src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -367,7 +369,7 @@ FindImplicitGemmWrwGTCDynamicXdlopsKernel(const ConvolutionContext& ctx)
                     gemm_n_per_block = 1 << r;
                 }
 
-                if(gemm_m % gemm_m_per_block != 0 || gemm_n % gemm_n_per_block != 0)
+                if(gemm_m % gemm_m_per_block != 0 || c % gemm_n_per_block != 0)
```

Comment:
I guess you'd better explain why here `c` need to check modulo of `gemm_n_per_block`, not `gemm_n = c*y*x`. because from the meaning of this symbol, `gemm_n` should evenly devided by `gemm_n_per_block`

Reply:
In our igemm_gen, here actually is (c%(gemm_n_per_block/nxe)!=0&(x*y)%nxe!=0). For another example, in BWD pass, the condition is n%(gemm_n_per_block/nxb)!=0&(ht*wt)%nxb!=0.
As we do not yet implement nxe greater than 1, so here gemm_n_per_block/nxe always equals to gemm_n_per_block. And (x*y)%nxe will always be true.

Reply:
I think you can not make this assumption that if `nxe` always be `1` then you are OK break the meaning of the original symbol, this may make code matainance difficult.

Again, by add following check is enough?
```
if( c  %  (  gemm_n_per_block / (nxe == 0?1:nxe)  )  != 0)
    return false;
```